spec/models/survey_messages/gyr_completion_survey_spec.rb

```diff
@@ -2,45 +2,66 @@
 
 RSpec.describe SurveyMessages::GyrCompletionSurvey do
```

Comment:
tried reorganizing these tests to make them more intuitive but the odd thing about them is that the thing being manipulated isn't the data in the setup (date of the tax return transition) but the argument being passed in (`expected_send_time` etc)

would also love another set of eyes on whether these cases are comprehensive enough